xbmcgui.py

```diff
@@ -338,8 +338,12 @@ def __str__(self):
             for key in keys:
                 value = "%s    - %s: %s\n" % (value, key, self.__properties[key])
 
+            if self.PY2:
+                return value.encode('utf-8') if isinstance(value, unicode) else value
```

Comment:
With PyCharm set to Python3, it does not like the unicode and shows it as error. Could we inverse the `if`?

```
return value if isinstance(value, bytes) else value.encode('utf-8')
```

and 

```
return self.__label if isinstance(self.__label, bytes) else self.__label.encode('utf-8')
```

I know it's small, but again, it makes my inner Sheldon happy :)

Reply:
Sure! Although it needs to be str instead of bytes iirc.